src/ast.ts

```diff
@@ -39,11 +40,11 @@ class ASTFeature implements vscodelc.StaticFeature {
         // clangd.ast.hasData controls the view visibility (package.json).
         adapter.onDidChangeTreeData((_) => {
           vscode.commands.executeCommand('setContext', 'clangd.ast.hasData',
-                                         adapter.hasRoot())
-          // Show the AST tree even if it's beet collapsed or closed.
+                                         adapter.hasRoot());
+          // Show the AST tree even if it's been collapsed or closed.
           // reveal(root) fails here: "Data tree node not found".
           if (adapter.hasRoot())
-          tree.reveal(null);
+            tree.reveal(null!);
```

Comment:
This one is kind of silly since we are asserting that null is not null. But, the API expects a non-null object, and if passing null works around the issue described, then this should be fine.

Reply:
I think maybe we should just @ts-ignore here instead, the current plan is for them to fix the typings: https://github.com/microsoft/vscode/commit/e95c40c1e461f83f5ff1c68d61253ab23c85f86b